examples/react/getting-started/index.html

```diff
@@ -9,15 +9,6 @@
 
     <link rel="shortcut icon" href="favicon.png" />
 
-    <!--
-      Do not use @8 in production, use a complete version like x.x.x, see website for latest version:
-      https://www.algolia.com/doc/guides/building-search-ui/installation/js/#load-the-styles
-    -->
-    <link
```

Comment:
the changes to example still have to be reverted before merging (if we merge before usage)